### DISCUSSION TITLE
HTML, CSS setup

### DISCUSSION THREAD

**Discussion by felipe300** [LazyVim/LazyVim#517] [General]
I'm new in programming and i'm just learning html, css and javascript. I usuallly use Vscode, but i want to try vim, is there a plugin or a workflow to use to speed my work? in Vscode i usually use "liveserver", but i don't know if there is in lazyvim

**Comment by 2giosangmitom** [comment]
You can install live-server package via npm

npm install live-server -g

**Comment by felipe300** [comment]
i 'll try that, thxnks!
